docker/config.ru

```diff
@@ -12,8 +12,8 @@ unless config.key?(:url)
   port = ENV.fetch('REDIS_PORT', 6379)
   db_number = ENV.fetch('REDIS_DB_NUMBER', 0)
 
-  credentials = ENV.fetch('REDIS_USERNAME', '')
-  credentials = "#{credentials}:#{ENV['REDIS_PASSWORD']}" if ENV.key?('REDIS_PASSWORD')
+  credentials = ERB::Util.url_encode(ENV.fetch('REDIS_USERNAME', ''))
```

Comment:
Thanks a lot for fixing the bug.

I would like to ask you to use `URI::DEFAULT_PARSER.escape` to escape url dangerous characters.

Reply:
Done!